runtime/src/bank/bank_hash_details.rs

```diff
@@ -162,6 +170,7 @@ impl TryFrom<&Bank> for SlotDetails {
             accounts_delta_hash,
             bank.signature_count(),
             bank.last_blockhash(),
+            bank.wait_get_epoch_accounts_hash(),
```

Comment:
Just making sure - this second call to `wait_get_epoch_accounts_hash()` will use a cached value and NOT recalculate. Based on my 30 second skim of the `Manager` struct, this appears to be the case. Doesn't hurt to double check tho 😄